packages/api/src/external/commonwell/patient.ts

```diff
@@ -76,44 +81,25 @@ export function getLinkStatusCQ(data: PatientExternalData | undefined): CQLinkSt
   return getCWData(data)?.cqLinkStatus ?? defaultStatus;
 }
 
-type StoreIdsAndStatusFunction = (params: {
-  commonwellPatientId: string;
-  personId?: string;
-  status?: LinkStatus;
-}) => Promise<void>;
-
-function getStoreIdsAndStatusFn(
-  patientId: string,
-  cxId: string,
-  cqLinkStatus?: CQLinkStatus
-): StoreIdsAndStatusFunction {
-  return async ({
-    commonwellPatientId,
-    personId,
-    status,
-  }: {
-    commonwellPatientId: string;
-    personId?: string;
-    status?: LinkStatus;
-  }): Promise<void> => {
-    await setCommonwellIdsAndStatus({
-      patientId,
-      cxId,
-      commonwellPatientId,
-      commonwellPersonId: personId,
-      commonwellStatus: status,
-      cqLinkStatus,
-    });
-  };
-}
-
-export async function create(
-  patient: Patient,
-  facilityId: string,
-  getOrgIdExcludeList: () => Promise<string[]>,
-  requestId?: string,
-  forceCWCreate = false
-): Promise<void> {
+export async function create({
+  patient,
+  facilityId,
+  getOrgIdExcludeList,
+  requestId,
+  forceCWCreate = false,
+  rerunPdOnNewDemographics = false,
+  callSynchronous = false,
+  initiator,
+}: {
+  patient: Patient;
+  facilityId: string;
+  getOrgIdExcludeList: () => Promise<string[]>;
+  requestId?: string;
```

Comment:
Im assuming this is optional now because you want each hie to have its own requestId. Looking at where this function is being called it doesnt seem like there is a need for this param anymore.